doc/getting-started.md

```diff
@@ -0,0 +1,151 @@
+## Getting Started
```

Comment:
This document is entererly new (I mean, not moved from some other in this PR)? Fine with it, just to know...

In addition, it should be linked from README.md.

Reply:
Readme updated. Fix c45ba05